scripts/irods/test/test_iadmin.py

```diff
@@ -2180,38 +2176,42 @@ def test_moduser_type_rodsadmin_rodsuser(self):
         self.admin.assert_icommand(['iadmin', 'moduser', self.username, 'type', 'rodsadmin'])
         self.assertEqual('rodsadmin', lib.get_user_type(self.admin, self.username))
 
-        # rodsadmin -> groupadmin
-        self.admin.assert_icommand(['iadmin', 'moduser', self.username, 'type', 'groupadmin'])
-        self.assertEqual('groupadmin', lib.get_user_type(self.admin, self.username))
-
         # groupadmin -> rodsuser
         self.admin.assert_icommand(['iadmin', 'moduser', self.username, 'type', 'rodsuser'])
         self.assertEqual('rodsuser', lib.get_user_type(self.admin, self.username))
 
-
     def test_moduser_type_rodsgroup(self):
         """Test modifying the user's type to a group (not allowed)."""
         self.assertEqual('rodsuser', lib.get_user_type(self.admin, self.username))
         self.admin.assert_icommand(['iadmin', 'moduser', self.username, 'type', 'rodsgroup'],
                                    'STDERR', 'SYS_NOT_ALLOWED')
         self.assertEqual('rodsuser', lib.get_user_type(self.admin, self.username))
 
-    def test_downgrade_of_service_account_user_is_not_allowed__6127(self):
-        self.assertEqual('rodsadmin', lib.get_user_type(self.admin, 'rods'))
-        desired_error_msg = "Cannot downgrade another rodsadmin"
-        self.admin.assert_icommand(['iadmin', 'moduser', 'rods', 'type', 'rodsuser'], 'STDERR_SINGLELINE', desired_error_msg)
-        self.assertEqual('rodsadmin', lib.get_user_type(self.admin, 'rods'))
-
-        self.admin.assert_icommand(['iadmin', 'moduser', 'rods', 'type', 'groupadmin', 'STDERR_SINGLELINE', desired_error_msg])
-        self.assertEqual('rodsadmin', lib.get_user_type(self.admin, 'rods'))
-
     def test_moduser_type_invalid_type(self):
         """Test modifying the user's type to something that is not supported."""
         self.assertEqual('rodsuser', lib.get_user_type(self.admin, self.username))
         self.admin.assert_icommand(['iadmin', 'moduser', self.username, 'type', 'invalid_user_type'],
                                    'STDERR_SINGLELINE', 'CAT_INVALID_USER_TYPE')
         self.assertEqual('rodsuser', lib.get_user_type(self.admin, self.username))
 
+    def test_downgrade_of_service_account_user_is_not_allowed__issue_6127(self):
+        """Test downgrading of service account user's type from rodsadmin to other supported types."""
+        # rodsadmin -> rodsuser
+        self.assertEqual('rodsadmin', lib.get_user_type(self.admin, 'rods'))
+        downgrade_error_msg = 'Cannot downgrade another rodsadmin'
```

Comment:
Should we include more words or a comment here? I got a little confused when I saw this expected message next to the title of the test. It makes it look like no `rodsadmin` is allowed to downgrade any other `rodsadmin`. Maybe that's just me :)

Reply:
Will use more words.

Reply:
Done